src/crypto/index.js

```diff
@@ -288,11 +295,19 @@ function _maybeUploadOneTimeKeys(crypto) {
 
     crypto._oneTimeKeyCheckInProgress = true;
     q().then(() => {
+        if (currentKeyCount !== undefined && isFinite(currentKeyCount)) {
+            // previously this parameter was not defined.
```

Comment:
the parameter is still not defined when this function is called from _onSyncCompleted.